football-club-management/src/test/java/org/footballclub/footballclubmanagement/config/TestConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.footballclub.footballclubmanagement.config;
+
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableJpaRepositories(basePackages = "org.footballclub.footballclubmanagement.repository")
+public class TestConfig {
+    @Bean
+    public DataSource dataSource(){
```

Comment:
avoid creating datasource bean by hand as this should be configured using standard spring jpa datasource properties in application properties or yaml file